Sample Data/Custom/BitglassLogs_CL.json

```diff
@@ -0,0 +1,215 @@
+[
+   {
+      "action":"",
+      "application":"Google Apps",
+      "fileid":"1WdkjMqjaVefVSwx-NXaD5_W2ZPJrtpTk",
```

Comment:
Parser is not working with given sample data. Please update fileid with proper guid format in sample data.
`XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX`
![image](https://user-images.githubusercontent.com/75478517/135387514-299914d5-7289-48f1-ad27-a7511bf5aaf6.png)

Reply:
done